aspnetcore/grpc/loadbalancing.md

```diff
@@ -109,14 +109,14 @@ This example creates a new <xref:Microsoft.Extensions.DependencyInjection.Servic
 
 ## Configure load balancer
 
-A load balancer is specified in a service config using the `ServiceConfig.LoadBalancingConfigs` collection. Two load balancers are built-in and map to load balancer config names:
+A load balancer is specified in a [service config](https://github.com/grpc/grpc/blob/master/doc/service_config.md) using the `ServiceConfig.LoadBalancingConfigs` collection. Two load balancers are built-in and map to load balancer config names:
 
 | Name          | Type                            | Description |
 | ------------- | ------------------------------- | ----------- |
 | `pick_first`  | `PickFirstLoadBalancerFactory`  | Attempts to connect to addresses until a connection is successfully made. gRPC calls are all made to the first successful connection. |
 | `round_robin` | `RoundRobinLoadBalancerFactory` | Attempts to connect to all addresses. gRPC calls are distributed across all successful connections using [round-robin](https://www.nginx.com/resources/glossary/round-robin-load-balancing/) logic. |
 
-There are a couple of ways a channel can get a service config with a load balancer configured:
+Service config is an abbreviation of service configuration and is represented by the `ServiceConfig` type. There are a couple of ways a channel can get a service config with a load balancer configured:
```

Comment:
The additional sentence breaks up the flow from the table to "There are a couple ways ...". Given the focus here on the load balancing config collection rather than on the service config itself, I would rely on the link rather than attempt to explain what it is inline.  

For someone new to grpc the linked doc could do a better job of explaining what it is -- "... is a mechanism..." isn't very descriptive.  It sounds like a collection of server settings but how you configure them and how they are stored (aside from JSON vs. protobuf form) isn't immediately clear from the Service Config doc.